go/libkb/notify_router.go

```diff
@@ -2644,6 +2646,25 @@ func (n *NotifyRouter) HandleHTTPSrvInfoUpdate(ctx context.Context, info keybase
 	})
 }
 
+func (n *NotifyRouter) HandleHandleKeybaseLink(ctx context.Context, link string) {
+	if n == nil {
+		return
+	}
+	n.cm.ApplyAll(func(id ConnectionID, xp rpc.Transporter) bool {
+		if n.getNotificationChannels(id).Service {
+			go func() {
+				_ = (keybase1.NotifyServiceClient{
+					Cli: rpc.NewClient(xp, NewContextifiedErrorUnwrapper(n.G()), nil),
+				}).HandleKeybaseLink(ctx, link)
+			}()
+		}
+		return true
+	})
```

Comment:
A couple of them have comments on why they are synchronized, most of them don't. I don't see any reason why these specific notifications would need synchronization, but I am happy to add a WaitGroup there as you suggest if you think it would make the code more robust....